Update dg v1.3 (#203)
AY2021S1-CS2103T-T10-4/tp: docs/UserGuide.md

Gives you an overall summary of a student's attendance and participation scores.

Format: `stats c\CLASS_INDEX s\STUDENT_INDEX`
* Gives a summary of the student's attendance at the specified `CLASS_INDEX` + `STUDENT_INDEX`.

For example:
* It is the end of the semester and time for you to grade a student's class participation. You would like to find out
 if Alex has been participative throughout the semester and hence you type in the command `stats c\1 s\1` and press
 <kbd>Enter</kbd>.
* Now you can see his average participation score and overall attendance to give a corresponding grade.

    ![Display_stats after](images/ugimages/DisplayStatsAfter.png)

<div markdown="span" class="alert alert-info">

**:information_source: Notes about the command:**<br>

>   * Lesson(s) not attended displays all lessons that the student did not attend. This means that future lessons 
>     would also be displayed in the results.

Reviewer: Would it be clearer write like this?
```suggestion
  * `Lesson(s) not attended` displays all lessons not attended by the student. This means that future lessons 
     are also displayed.
```

## Glossary

> This glossary aims to provide a definition for the special vocabulary used in this user guide.

Reviewer: ```suggestion
This glossary provides definitions for special terms used in this guide.
```
